manager/gui/src/components/dashboard/LrDashboardPerformanceCard.vue

```diff
@@ -51,12 +78,12 @@ export default class LrDashboardPerformanceCard extends Vue {
   public dashboardCardDetails: DashboardCardDetail[] = [];
 
   public async mounted() {
-    const data = (await Api.getData("/dashboard/performance", {
+    const data: any = await Api.getData("/dashboard/performance", {
```

Comment:
I would suggest to get all the data from dummy json as 'data'. So that we can call 'res.data' in our code. Because, this is how we are going to access data from the actual API response.

Reply:
Changes done to data.